src/main/java/gov/usds/case_issues/services/CaseListService.java

```diff
@@ -306,7 +306,8 @@ public DataFormatSpec getUploadFormat(String uploadFormatId) {
 		Function<? super Object[], ? extends CaseSummary> mapper = row ->{
 			TroubleCase rootCase = (TroubleCase) row[0];
 			ZonedDateTime lastSnoozeEnd = (ZonedDateTime) row[1];
-			CaseSnoozeSummary summary = lastSnoozeEnd == null ? null : _snoozeRepo.findFirstBySnoozeCaseOrderBySnoozeEndDesc(rootCase).get();
+			CaseSnoozeSummary summary = lastSnoozeEnd == null ? null
```

Comment:
😿

Reply:
Is this a complaint about formatting?

Reply:
Not so much about formatting per se as about formatting changes in what is now an otherwise unchanged method. Not exactly a crisis, I realize, it just made me mildly sad.